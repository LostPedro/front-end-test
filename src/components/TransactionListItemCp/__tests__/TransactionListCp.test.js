import React from 'react'
import {render, cleanup} from '@testing-library/react'
import {TransactionListItemCp} from '../'

afterEach(cleanup)

it('should take a snapshot', () => {
  const {asFragment} = render(<TransactionListItemCp />)

  expect(asFragment(<TransactionListItemCp />)).toMatchSnapshot()
})
