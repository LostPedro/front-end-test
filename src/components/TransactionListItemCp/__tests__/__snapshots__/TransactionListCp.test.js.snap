// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="transaction-list-item-component"
  >
    <div
      class="transaction-list-item-component-item-text-line"
    >
      <span
        class="transaction-list-item-component-name"
      />
      <span
        class="transaction-list-item-component-status"
      />
    </div>
    <div
      class="transaction-list-item-component-item-text-line"
      style="margin-top: 8px;"
    >
      <span
        class="transaction-list-item-component-date"
      />
      <span
        class="transaction-list-item-component-money-amount"
      />
    </div>
  </div>
</DocumentFragment>
`;
